pyblish_lite/app.py

```diff
@@ -52,9 +55,23 @@ def show(parent=None):
 
         ctrl = control.Controller()
 
-        win = window.Window(ctrl, parent)
-        win.resize(430, 600)
-        win.show()
+        win = None
+        try:
+            win = self._window
+            win.show()
+            win.activateWindow()
+            win.setWindowTitle(settings.WindowTitle)
+            print "Using existing window..."
+        except:
```

Comment:
There are too many potential exceptions here, better except with specific errors in mind.

Also, some of those things don't need a try/except; like setting the Window title. Keep it minimal.